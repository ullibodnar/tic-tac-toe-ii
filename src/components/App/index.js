import React from 'react'

import { Header } from '..'
import { Settings, Game } from '../../containers'

export default function App () {
  return (
    <div>
      {/* outer div */}

      <Header name={'Tic-Tac-Toe Mk. II'} />

      <Game />

      <Settings />

      {/* outer div */}
    </div>
  )
}
