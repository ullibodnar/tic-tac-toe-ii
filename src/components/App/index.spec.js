import React from 'react'
import { shallow } from 'enzyme'

import App from '.'

describe('components:App', () => {
  it('renders the App with a blank game board and nine squares', () => {
    expect(toJson(shallow(<App />))).toMatchSnapshot()
  })

  it('renders the App with a game board three moves: center, top-left, top-right', () => {
    expect(toJson(shallow(<App moves={[4, 0, 2]} />))).toMatchSnapshot()
  })
})
