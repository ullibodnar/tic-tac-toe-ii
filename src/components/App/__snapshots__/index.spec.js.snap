// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:App renders the App with a blank game board and nine squares 1`] = `
<div>
  <Header
    name="Tic-Tac-Toe Mk. II"
  />
  <Game />
  <Settings />
</div>
`;

exports[`components:App renders the App with a game board three moves: center, top-left, top-right 1`] = `
<div>
  <Header
    name="Tic-Tac-Toe Mk. II"
  />
  <Game />
  <Settings />
</div>
`;
